client/landing/stepper/hooks/use-site-copy.tsx

```diff
@@ -77,6 +77,7 @@ export const useSiteCopy = (
 			return;
 		}
 		clearSignupDestinationCookie();
+		resetOnboardStore();
```

Comment:
We can probably achieve the same goal using `setDomainSearch` which can be less intrusive for other flows.

Reply:
Using `setDomainSearch` will also make more evident the intention than calling `resetOnboardStore` broadly.

Reply:
@sejas I actually tried `setDomainSearch()` originally but it didn't have the impact I wanted. Also, we probably want cart information, etc. cleared too.